common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -218,22 +218,17 @@ def student_view(self, context):
             rendered_child = child.render(STUDENT_VIEW, context)
             fragment.add_frag_resources(rendered_child)
 
-            # `titles` is a list of titles to inject into the sequential tooltip display.
-            # We omit any blank titles to avoid blank lines in the tooltip display.
-            titles = [title.strip() for title in child.get_content_titles() if title.strip()]
             childinfo = {
                 'content': rendered_child.content,
-                'title': "\n".join(titles),
-                'page_title': titles[0] if titles else '',
+                'page_title': child.tooltip_content if child.tooltip_content else '',
```

Comment:
Change this to: `getattr(child, 'tooltip_content', '')` in case the child doesn't have a `tooltip_content` property.